zebra-state/src/service.rs

```diff
@@ -1554,6 +1557,60 @@ impl Service<ReadRequest> for ReadStateService {
                 .map(|join_result| join_result.expect("panic in ReadRequest::BestChainBlockHash"))
                 .boxed()
             }
+
+            // Used by get_block_template RPC.
+            #[cfg(feature = "getblocktemplate-rpcs")]
+            ReadRequest::ChainInfo() => {
+                metrics::counter!(
+                    "state.requests",
+                    1,
+                    "service" => "read_state",
+                    "type" => "chain_info",
+                );
+
+                let timer = CodeTimer::start();
+
+                let state = self.clone();
+                let network = self.network;
```

Comment:
Nitpick: we could use `state.network` below and prune this line.

Reply:
Thanks, i did this in https://github.com/ZcashFoundation/zebra/pull/5659/commits/e80514a675a1c278b8c01d3b072ca68164d1fa0d